src/sinks/splunk_hec/logs/config.rs

```diff
@@ -54,23 +64,35 @@ pub struct HecLogsSinkConfig {
     ///
     /// [splunk_field_index_docs]: https://docs.splunk.com/Documentation/Splunk/8.0.0/Data/IFXandHEC
     #[serde(default)]
+    #[configurable(metadata(docs::examples = "field1", docs::examples = "field2"))]
     pub indexed_fields: Vec<String>,
 
     /// The name of the index where to send the events to.
     ///
     /// If not specified, the default index is used.
+    #[configurable(metadata(docs::examples = "{{ host }}", docs::examples = "custom_index"))]
     pub index: Option<Template>,
 
     /// The sourcetype of events sent to this sink.
     ///
     /// If unset, Splunk will default to `httpevent`.
+    #[configurable(metadata(
+        docs::examples = "{{ sourcetype }}",
+        docs::examples = "_json",
+        docs::examples = "httpevent"
+    ))]
     pub sourcetype: Option<Template>,
```

Comment:
same, and can remove the "httpevent" from the examples since it would be the default.